src/components/ProjectsPage/ProjectsMonitoring/ProjectsMonitoring.js

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright 2019 Iguazio Systems Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License") with
+an addition restriction as set forth herein. You may not use this
+file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+In addition, you may not use the software for any purposes that are
+illegal under applicable law, and the grant of the foregoing license
+under the Apache 2.0 license is conditioned upon your compliance with
+such restriction.
+*/
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import moment from 'moment'
+
+import DatePicker from '../../../common/DatePicker/DatePicker'
+import Loader from '../../../common/Loader/Loader'
+import StatsCard from '../../../elements/StatsCard/StatsCard'
+
+import jobsActions from '../../../actions/jobs'
+import workflowActions from '../../../actions/workflow'
+
+import { GROUP_BY_WORKFLOW, REQUEST_CANCELED, STATE_FILTER_ALL_ITEMS } from '../../../constants'
+import { getJobsStatsConfig } from '../projects.util'
+
+const ProjectsMonitoring = () => {
+  const [jobsFilter, setJobsFilter] = useState({
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    }
+  })
+  const [workflowsFilter, setWorkflowsFilter] = useState({
+    groupBy: GROUP_BY_WORKFLOW,
+    dates: {
+      value: [new Date(moment().add(-1, 'days'))]
+    },
+    state: STATE_FILTER_ALL_ITEMS
+  })
+  const [scheduledFilter, setScheduledFilter] = useState({
+    dates: {
+      value: [new Date(), new Date(moment().add(1, 'days'))]
+    }
+  })
+  const [loadingState, setLoadingState] = useState({
+    jobs: true,
+    workflows: true,
+    scheduled: true
+  })
+
+  const dispatch = useDispatch({})
+  const navigate = useNavigate({})
+  const jobsAbortControllerRef = useRef()
+  const workflowsAbortControllerRef = useRef()
```

Comment:
we need abortController also for Scheduled card. The request should be canceled on component unmount